apps/web/src/pages/integrations/IntegrationsStoreModal.tsx

```diff
@@ -175,7 +182,7 @@ export function IntegrationsStoreModal({
           classNames={drawerClasses}
         >
           <IntegrationCardWrapper>
-            <When truthy={!provider?.novu}>
+            <When truthy={!provider?.novu && provider?.providerId !== InAppProviderIdEnum.Novu}>
```

Comment:
Is it possible to make in app also have the novu field as true?

Reply:
Right now it is not because it needs to be more custom :(